src/os/bluestore/BlueStore.cc

```diff
@@ -5094,6 +5113,11 @@ int BlueStore::_open_alloc()
     return -EINVAL;
   }
 
+  if (bdev->is_smr()) {
+    min_alloc_size &= 0x00000000ffffffff;
```

Comment:
You might want to remove similar stuff reverting min_alloc_size immediately after Allocator::create call.
Or just omit min_alloc_size spoiling and use another local variable when passing to Allocator::create. Hence no need to revert...

Reply:
Done.  Used another variable and moved it to a function to avoid duplication.